packages/connect-explorer/src/components/Settings.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components';
+
+import { Button } from '@trezor/components';
+
+import * as trezorConnectActions from '../actions/trezorConnectActions';
+import { useSelector, useActions } from '../hooks';
+
+import { Row } from './fields/Row';
+import { getField } from './Method';
+import { Field, FieldWithBundle } from '../types';
+
+const SettingsContent = styled.section`
+    flex: 1;
+    padding: 10px 20px;
+    display: flex;
+    flex-direction: column;
+`;
+
+export const Settings = () => {
+    const connectOptions = useSelector(state => ({
+        trustedHost: state.connect?.options?.trustedHost || false,
```

Comment:
in general, trusted host should only be "downgradable", so this line feels correct. I am just not sure whether this option is taken into account by connectSettings.ts. is it?

Reply:
Yes it is. Done by the action `onConnectOptionChange` which is passed as `onFieldChange` to the field.

I wanted to reuse the existing mechanisms and it works.

If there is more time to improve connect-explorer code I would like to refactor it a bit the `GetField`  but for now it works.